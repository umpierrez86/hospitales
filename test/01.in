leer_instancia
1 1
1
0
0
imprimir_instancia
asignar_residencias
imprimir_asignacion
Fin