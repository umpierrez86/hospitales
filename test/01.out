1>2>Cantidad hospitales: 1
Cupos hospitales: 1 
Cantidad estudiantes: 1
Preferencias hospitales
Preferencias hospital h0: 0 
Preferencias estudiantes
Preferencias estudiante e0: 0 
3>4>(0, 0) 
5>Fin.
